test/unit/test_initial_condition.py

```diff
@@ -19,3 +20,32 @@ def test_error_initialisation_from_xdmf_missing_label():
     """
     with pytest.raises(ValueError, match=r"label"):
         festim.InitialCondition(value="my_file.xdmf", label=None, time_step=1)
+
+
+def test_temperature_and_DirichletBC():
```

Comment:
I suppose this test is similar to the one above. Moreover, it's not really related to inital conditions.